client/src/Controller/Admin/CaseController.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace App\Controller\Admin;
+
+use App\Controller\AbstractController;
+use App\Service\Client\RestClient;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Security;
+use Symfony\Component\Routing\Annotation\Route;
+
+/**
+ * @Route("/admin/case")
```

Comment:
Similar to above comment - could we get this prepended to the actual route below?